apps/aehttp/priv/swagger.yaml

```diff
@@ -1665,15 +1705,6 @@ paths:
 ##
 
 parameters:
-  HeightIn:
```

Comment:
Why is it an improvement to inline this? Just wondering

Reply:
Old behaviour was that this was inlined for us by our tools that produced [swagger.json](https://github.com/aeternity/aeternity/blob/v4.2.0/apps/aehttp/priv/swagger.json#L224-L232). Since we depricated those tools for good, now suddenly we don't inline this and it causes issues client-side.

Not really improvement but more of returning to initial behaviour from client's perspective